README.md

```diff
@@ -118,7 +157,14 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+An ERD (entity-relationship diagram) gives an overview of your models and how they are associated.
+
+The attributes and relationships for the example ERD might be . . .
+
+- Genie: wishes, amount
+- Lamp: material, shape, weight
+- Person: wishes, hope, amount
+- Pet: animal, type,
 ```
 
```

Comment:
An ERD -- or "Entity Relationship Diagram" -- is a visual tool we use to identify the data values being saved in an application, groupings of those data values and the relationships between them.

Sample relationships...
- A one-to-one relationship between Genie and Lamp
- A one-to-many relationship between Person and Pet